### DISCUSSION TITLE
Thank you for this ytdlp GUI.

### DISCUSSION THREAD

**Discussion by KonaKona7** [ErrorFlynn/ytdlp-interface#87] [General]
It's literally the only sane one that exists.
Prefix numbering for playlists by default.
Folders per playlist by default.
All naming scheme options as checkboxes instead of requiring end user to learn some CLI template formatting.
A dark mode that looks good and is actually complete instead of half-assed.
C++ instead of some ugly bloated Electron node.js monstrosity.
Everything is perfect out of the box and doesn't even require any configuring for 99% of use cases.
Common sense and attention to detail are rare finds among most projects, but yours has both in spades.
You are a gentleman and a scholar, and a Starcraft fan to boot.

**Comment by ErrorFlynn** [comment]
Thank you, I appreciate that. Many of those features were requested by users, and I did my best to implement them. Unfortunately I'm no longer able to continue working on the project, at least for now. Maybe some day I'll be able to resume, but either way, I'm happy with how it turned out.

**Reply by Spottedfeather** [reply]
I'm sorry, but it doesn't work.

**Comment by tyrell-wallace** [comment]
Yeah, this is best-in-class. It also outputs files with more metadata than other yt-dlp frontends I've tested.

**Comment by QuailRider43** [comment]
Yes, thank you for writing this software. My old yt-dlp gui client became outdated and stopped working. Searching for a replacement, I found ytdlp-interface. Works perfectly, portable, easy to update to the latest versions. Love it. Grateful for your efforts!!
